### PR TITLE
Mentorship new branch

### PR DESCRIPTION
**Pull request by Chinonyemildred**
## What did you do? 
- I edited the mentorship_guidelines.md file to mentorship.md
- I created the mentorship.vue page and added the mentorship training video and resource link
-I also added a link from the mentorship page to the nav bar

## Why did you make this change?
I switched branch from this #28  because of the failed test

## Have you tested locally?
(please check all three to move your PR forward!)
- [x] I have tested the changes locally and they appear as I desire them to.
- [x] I have tested the links and they work correctly.
- [x] I have checked that the images match my expectations.

## How to test the changes? 
Instructions for manual testing are as follows:

1. Run `npm run develop`

## Add a screenshot if applicable
Screenshot 2022-02-11 202106

### CODE REVIEW

File: src/pages/Mentorship.vue [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+            <h1 class="page-title">{{ $page.main.title }}</h1>
+            <div class="markdown" v-html="$page.main.content" /> 
+         </b-col>
+      <b-col lg="5" md="5">
+
+          <h2>Mentorship Training</h2>
+          <p>We encourage all participants to watch this video on open source mentorship.</p>
+          <iframe width="100%" height="314" src="https://www.youtube.com/embed/6W5najMfcy0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          <p>Video credit: MongoDB</p>
+          <p>More information to guide you can be found in this <a href="https://www.mentorship.guide/">resource</a></p>
```
Comment (Chinonyemildred) [inline src/pages/Mentorship.vue:14]:
```suggestion
          <p>More information to guide you can be found in this <a href="https://www.mentorship.guide/" target="_blank">resource</a></p>
```

File: content/mentorship.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
-
-# Galaxy Mentorship Network Guidelines
+---
+id: mentorship
+title: GMN Mentorship Guidelines
```
Comment (bebatut) [inline content/mentorship.md:3]:
```suggestion
title: Mentorship Guidelines
```

File: src/pages/Mentorship.vue [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+         </b-col>
+      <b-col lg="5" md="5">
+
+          <h2>Mentorship Training</h2>
+          <p>We encourage all participants to watch this video on open source mentorship.</p>
+          <iframe width="100%" height="314" src="https://www.youtube.com/embed/6W5najMfcy0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+          <p>Video credit: MongoDB</p>
+          <p>More information to guide you can be found in this <a href="https://www.mentorship.guide/" target="_blank">resource</a></p>
+
+      </b-col>
```
Comment (bebatut) [inline src/pages/Mentorship.vue:16]: I am not convinced by having the 2 columns there. I would put it directly in the Markdown, so we can have on the right column the Table of content

"Mentorship Training" may sounds like it is only for mentors

Comment (Chinonyemildred) [reply src/pages/Mentorship.vue:16]: Maybe 'Participants' Training' would be better?

Comment (bebatut) [reply src/pages/Mentorship.vue:16]: I will have a look how to add the table of content for all pages

Comment (Chinonyemildred) [reply src/pages/Mentorship.vue:16]: > I will have a look how to add the table of content for all pages

Thank you!

File: src/pages/Mentorship.vue [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+         <b-row>
+      <b-col lg="7" md="7">
+            <h1 class="page-title">{{ $page.main.title }}</h1>
+            <div class="markdown" v-html="$page.main.content" /> 
+         </b-col>
+      <b-col lg="5" md="5">
+
+         
+
+      </b-col>
```
Comment (bebatut) [inline src/pages/Mentorship.vue:12]: Can you put back here as the other pages?

Comment (Chinonyemildred) [reply src/pages/Mentorship.vue:12]: I should take out the columns, right?

Comment (bebatut) [reply src/pages/Mentorship.vue:12]: And the row

File: src/pages/Mentorship.vue [hunk 5]
```diff
@@ -0,0 +1,30 @@
+<template>
+     <Layout>
+   
+            <h1 class="page-title">{{ $page.main.title }}</h1>
+            <div class="markdown" v-html="$page.main.content" /> 
+        
+     </Layout>
```
Comment (bebatut) [inline src/pages/Mentorship.vue:7]:
```suggestion
     <Layout>
            <h1 class="page-title">{{ $page.main.title }}</h1>
            <div class="markdown" v-html="$page.main.content" /> 
     </Layout>
```

**Pull request merged**